src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
Even if you look at the specification here: https://notifications.spec.whatwg.org/

There is no mention whatsoever to the Push API. They are completely independent